### PR TITLE
fix camel case averageRtcpInterval and also put in RTCInboundRTPStreamStats

### PR DESCRIPTION
**Pull request by vr000m**
Fixes issue #305. However #272 introduced a bug that the rtcp interval is only available at the sending side and not the receiver side. this fixes that issue.

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -1381,7 +1394,7 @@ <h3>
              double               targetBitrate;
              unsigned long        framesEncoded;
              double               totalEncodeTime;
-             double               averageRTCPInterval;
+             double               averageRtcpInterval;
```
Comment (alvestrand) [inline webrtc-stats.html]: Please also add the old spelling to the "obsolete stats" section. It's been burned.

Comment (vr000m) [reply webrtc-stats.html]: was it implemented as yet by anyone? @jan-ivar @aboba and will mention that it was originally  in RTCRTPStreamStats

Comment (vr000m) [reply webrtc-stats.html]: fixed, PTAL @alvestrand

**Comment by vr000m** [comment]
Need to fix travis comments.

File: webrtc-stats.html (hunk 1 above)
Comment (jan-ivar) [reply webrtc-stats.html]: We haven't implemented it.

**Pull request merged**